setup.py

```diff
@@ -455,6 +464,88 @@ def run(self):
         if self._ran_build:
             return
 
+        try:
+            # Following code is for building the Qualcomm backend.
+            from backends.qualcomm.scripts.download_qnn_sdk import (
+                _download_qnn_sdk,
+                SDK_DIR,
+            )
+
+            os.environ["EXECUTORCH_BUILDING_WHEEL"] = "1"
+
+            # qnn sdk setup
+            _download_qnn_sdk()
+
+            sdk_path = Path(SDK_DIR).resolve()  # full absolute path
```

Comment:
Can you install sdk in a temp directory so that you dont need to do explicit cleanup?

Reply:
updated to use temp directory